src/http3wtstreamvisitor.cc

```diff
@@ -69,7 +69,10 @@ namespace quic
     void Http3WTStream::doCanRead()
     {
         if (pause_reading_)
```

Comment:
may be add `&& !(sequencer_->IsClosed() && !fin_read_)` but only if we want that a fin unblocks the stream.
Anyway for both, if you can provoke the problem, a test would be good.